agent_dir/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+
+services:
+  agent:
+    build:
+      context: newrelic-infra-agent
+    cap_add:
+      - SYS_PTRACE
```

Comment:
is this added by the agent?

Reply:
Yeah, https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/linux-installation/docker-container-infrastructure-monitoring/